src/components/resource/HelpTooltip.jsx

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+import React from 'react/addons';
```

Comment:
`react/addons` is a special build of React that ships with additional modules (like `TestUtils` or `TransitionGroup`). Only require it when really needed, otherwise just require `react`.